template/src/core/app/appState.ts

```diff
@@ -0,0 +1,6 @@
+const appState = {
+    hasLoggedIn: false
+  };
+  
+  export {appState};
```

Comment:
```suggestion
const initialAppState = {
    hasLoggedIn: false
  };
```
I think we can rename this and move inside `appStateReducer.ts` for simplicity